<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UPnPTestWidget</class>
 <widget class="QWidget" name="UPnPTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UPnP Test Application</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Router:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_router">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="text">
          <string>No routers found.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="m_find_routers">
       <property name="text">
        <string>Find Routers</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="m_text_output"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_port">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>4000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Protocol:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_protocol">
         <item>
          <property name="text">
           <string>UDP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TCP</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_forward">
       <property name="text">
        <string>Forward</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_undo_forward">
       <property name="text">
        <string>Undo Forward</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_verbose">
       <property name="text">
        <string>Verbose mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
